assets/app/scripts/directives/catalog.js

```diff
@@ -4,28 +4,27 @@ angular.module('openshiftConsole')
   .directive('catalogTemplate', function($location) {
     return {
       restrict: 'E',
+      // Replace the catalog-template element so that the tiles are all equal height as flexbox items.
+      // Otherwise, you have to add the CSS tile classes to catalog-template.
+      replace: true,
       scope: {
         template: '=',
         project: '='
       },
       templateUrl: 'views/catalog/_template.html',
       link: function(scope, elem, attrs) {
-        $(".select-template", elem).click(function() {
-          // Must trigger off of the modal's hidden event to guarantee modal has finished closing before switching screens
-          $(".modal", elem).on('hidden.bs.modal', function () {
-            scope.$apply(function() {
-              var createURI = URI.expand("project/{project}/create/fromtemplate{?q*}", {
-                project: scope.project,
-                q: {
-                  name: scope.template.metadata.name,
-                  namespace: scope.template.metadata.namespace
-                }
-              });
-              $location.url(createURI.toString());
+        // TODO: use ng-click?
+        elem.click(function() {
```

Comment:
if at all possible, I'd like to make this URL calculation from a template a function or filter we can call in the view, so we can use ng-href and allow shift+clicking to launch in a new window, rather than handling with onclick